docs/CONTRIBUTING.md

```diff
@@ -64,11 +64,28 @@ t.Run("with nested changes trigger", func (t *testing.T) {
 
 **Note**: After your PR has been merged, and the feature either reaches general availability, you should remove the `skipIfBeta()` flag.
 
-## Best Practices for Adding a New Endpoint
+## Adding New Endpoints
 
-Here you will find a scaffold to get you started when building a json:api RESTful endpoint. The comments are meant to guide you but should be replaced with endpoint-specific and type-specific documentation. Additionally, you'll need to add an integration test that covers each method of the main interface.
+### Scaffolding a Resource
 
-In general, an interface should cover one RESTful resource, which sometimes involves two or more endpoints. Add all new modules to the tfe package.
+When creating a new resource you can use the helper script `generate_resource.sh` to quickly setup boilerplate code for adding a new set of endpoints related to that resource:
+
+```sh
+./generate_resource.sh organization
```

Comment:
suggesting to use: `./scripts/generate_resource.sh organization` in the instruction, while following the instruction, my first intuition was to run it in the resources folder as opposed to going into the script folder